restock.lic

```diff
@@ -47,13 +48,16 @@ class Restock
         reget(2, 'runestone') ? DRCI.put_away_item?(item['name'], @runestone_storage) : stow_hands
       end
     end
+    deposit_coins_from_specific_bank?(@keep_copper, @settings, town)
```

Comment:
What do you think about guarding this call on `town == "Shard"`?

Reply:
You mean switch to /\bShard\b/?